src/WebJobs.Script.WebHost/host.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "2.0",
+  "extensionBundle": {
+    "id": "Microsoft.Azure.Functions.ExtensionBundle",
+    "version": "[2.*, 3.0.0)"
+  }
+}
```

Comment:
You'll probably want to delete this file from the PR.

Reply:
Ah just removed this